.vscode/settings.json

```diff
@@ -58,7 +58,7 @@
 		"zustand"
 	],
 	"[typescript]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint"
+		"editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
if you can install and config the classing eslint (dbaumer one) that should fix the lint not being auto run